tests/src/test/java/alluxio/IntegrationTestUtils.java

```diff
@@ -30,6 +31,34 @@
  * Util methods for writing integration tests.
  */
 public final class IntegrationTestUtils {
+  public static boolean isGcs(UnderFileSystem ufs) {
```

Comment:
I think these utility methods should go into `alluxio.util.UnderFileSystemUtils` in `alluxio-core-common`